packages/eas-json/src/build/schema.ts

```diff
@@ -22,6 +22,7 @@ const CommonBuildProfileSchema = Joi.object({
   yarn: Joi.string().empty(null).custom(semverCheck),
   expoCli: Joi.string().empty(null).custom(semverCheck),
   env: Joi.object().pattern(Joi.string(), Joi.string().empty(null)),
+  autoIncrement: Joi.alternatives().try(Joi.boolean(), Joi.string().valid('version')),
```

Comment:
```suggestion
  autoIncrement: Joi.boolean(),
```

Reply:
To make `concat` on schema work I used `Joi.alternatives().try(Joi.boolean())`, because otherwise Joi wasn't able to merge `Joi.boolean` and `Joi.alternatives`